lib/board.rb

```diff
@@ -0,0 +1,62 @@
+class Board
+
+  attr_accessor :cells
+
+  def initialize()
+    #set attribute :cells 
+    #equal to our 'board' array with 9 empty spaces
+    @cells = Array.new(9," ")
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.